src/zappy_server_src/core/include/scheduler/scheduler.h

```diff
@@ -0,0 +1,54 @@
+/*
+** EPITECH PROJECT, 2022
+** ZAAAAAAAAAAAAppy
+** File description:
+** scheduler
+*/
+
+#ifndef SCHEDULER_H
+    #define SCHEDULER_H
+
+    #include <uuid/uuid.h>
+    #include <sys/queue.h>
+    #include <stdbool.h>
+
+/// \brief represent a scheduler event that will be stored inside the scheduler
+typedef struct scheduler_event_s {
+    /// \brief The entity uuid bound to the event
+    uuid_t id;
+
+    /// \brief The amount of time to wait before the event is over
+    int ticks;
+
+    TAILQ_ENTRY(scheduler_event_s) events;
+} scheduler_event_t;
+
+TAILQ_ENTRY(scheduler_event_s) scheduler_event_head;
```

Comment:
missing documentation (Check all the .h file, I won't wrote "missing documentation" each time)